aws/resource_aws_dlm_lifecycle_policy.go

```diff
@@ -106,9 +106,29 @@ func resourceAwsDlmLifecyclePolicy() *schema.Resource {
 											Schema: map[string]*schema.Schema{
 												"count": {
 													Type:         schema.TypeInt,
-													Required:     true,
+													Optional:     true,
+													ForceNew:     true,
 													ValidateFunc: validation.IntBetween(1, 1000),
```

Comment:
Just saw your comment above but in this case everything in the DLM API that is a list is also required to be 1 element in the list (AWS sure do like future proofing things at the expense of more complex APIs) so you can just hardcode the list element as 0 as in my comment here.

Reply:
I just realised that that's not currently true because `schedule` is missing a `MaxItems: 1` despite that being [required by the API](https://docs.aws.amazon.com/dlm/latest/APIReference/API_PolicyDetails.html) which I think I missed in my initial pull request that added this resource.

We should add that in this pull request I think. It then makes the `ConflictsWith` robust as well as giving plan time validation in the event someone adds multiple `schedule` blocks to a `policy_details` block.